packages/network/src/logic/trackerSummaryUtils.ts

```diff
@@ -1,8 +1,9 @@
-import { StreamIdAndPartition } from '../identifiers'
+import { Location, StreamIdAndPartition } from '../identifiers'
 import { OverlayPerStream, OverlayConnectionRtts } from './Tracker'
 
 type OverLayWithRtts = { [key: string]: { [key: string]: { neighborId: string, rtt: number | null }[] } }
 type OverlaySizes = { streamId: string, partition: number, nodeCount: number }[]
+type NodesWithLocations = { [key: string]: Location | null }[]
```

Comment:
Would a more tighter type be?
```suggestion
type NodesWithLocations = { [key: string]: Location }[]
```

Since below in the implementation of `getNodesWithLocationData` if a location is not found we use default value
```
{
                latitude: null,
                longitude: null,
                country: null,
                city: null,
            }
```